### ISSUE TITLE
Comment Bot

### ISSUE THREAD

**Issue by kordianbruck** [TCA-Team/TumCampusApp#587]
Parent Issue: #356
similar to the iOS repo, we would want errors directly displayed here in github PRs by the tca-bot

**Comment by psukys** [comment]
@kordianbruck to be more specific about the comment bot, what errors/information would it display?

**Comment by kordianbruck** [comment]
@psukys the bot should digest reports from the jenkins build and directly comment the code here on the github PR. 
* https://github.com/tomasbjerre/violation-comments-to-github-lib
* https://github.com/jenkinsci/violation-comments-to-github-plugin

Second there should be a bot that helps with PRs: 
https://github.com/voyagegroup/popuko

Please do a quick analysis of what is feasible to implement in max 2-3 hours. I can grant you access to our jenkins.

**Comment by psukys** [comment]
Looks pretty feasible with the plugin, just need to setup it with github to look through further. Maybe having #725 and #724 resolved first should be better, cause then it would give a bit more insight.

**Comment by psukys** [comment]
Alright, I think I got it running. I checked the repository for the commenter and parsers [can be nicely added](https://github.com/tomasbjerre/violations-lib/tree/master/src/main/java/se/bjurr/violations/lib/parsers). The project as well seems to be maintained and sort of active, which is nice.

Currently I can see that PMD and CHECKSTYLE reports are parsed, not sure about ANDROIDLINT one, time should show, but I think that this looks like nice start. I believe the issue can be set as resolved @kordianbruck
